src/main/webapp/add_release.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
Unrelated changes in one PR. Please split into multiple PRs.

Reply:
Also, this page is already in the master branch actually. You might want to sync your branch to the latest changes and/or diff against the right branch.

Reply:
I thought the changes were related because they are page adjustments to the add release page. I am also merging to master.